s3conf/storages.py

```diff
@@ -6,6 +6,7 @@
 
 import boto3
 from botocore.exceptions import ClientError
+from google.cloud import storage
```

Comment:
Maybe it is time to provide extras for this package. Is not nice to force people to install Google to use AWS or to install AWS to use Google.

Reply:
Completely agree, will do that in another PR though :+1: